tests/test_setbacks.py

```diff
@@ -372,13 +402,139 @@ def test_partial_exclusions_upscale_factor_less_than_1(mult):
     assert np.isclose(exclusion_mask, inclusion_weights).all()
 
 
-def test_cli_railroads(runner):
+@pytest.mark.parametrize(
+    ('setbacks_class', 'features_path', 'regulations_fpath',
+     'generic_sum', 'local_sum', 'setback_distance'),
+    [(StructureWindSetbacks,
+      os.path.join(TESTDATADIR, 'setbacks', 'RhodeIsland.gpkg'),
+      REGS_GPKG, 332_887, 142, [HUB_HEIGHT, ROTOR_DIAMETER]),
+     (RailWindSetbacks,
+      os.path.join(TESTDATADIR, 'setbacks', 'Rhode_Island_Railroads.gpkg'),
+      REGS_GPKG, 754_082, 9_402, [HUB_HEIGHT, ROTOR_DIAMETER]),
+     (ParcelSetbacks,
+      os.path.join(TESTDATADIR, 'setbacks', 'RI_Parcels', 'Rhode_Island.gpkg'),
+      PARCEL_REGS_FPATH_VALUE, 438, 3, [BASE_SETBACK_DIST]),
+     (WaterSetbacks,
+      os.path.join(TESTDATADIR, 'setbacks', 'Rhode_Island_Water.gpkg'),
+      WATER_REGS_FPATH_VALUE, 88_994, 83, [BASE_SETBACK_DIST])])
+@pytest.mark.parametrize('sf', [None, 10])
+def test_merged_setbacks(setbacks_class, features_path, regulations_fpath,
+                         generic_sum, local_sum, setback_distance, sf):
+    """ Test merged setback layers. """
+
+    generic_setbacks = setbacks_class(EXCL_H5, *setback_distance,
+                                      regulations_fpath=None, multiplier=100,
+                                      weights_calculation_upscale_factor=sf)
+    generic_layer = generic_setbacks.compute_setbacks(features_path,
+                                                      max_workers=1)
+
+    with tempfile.TemporaryDirectory() as td:
+        regs_fpath = os.path.basename(regulations_fpath)
+        regs_fpath = os.path.join(td, regs_fpath)
+        shutil.copy(regulations_fpath, regs_fpath)
+
+        local_setbacks = setbacks_class(EXCL_H5, *setback_distance,
+                                        regulations_fpath=regs_fpath,
+                                        weights_calculation_upscale_factor=sf,
+                                        multiplier=None)
+
+        local_layer = local_setbacks.compute_setbacks(features_path,
+                                                      max_workers=1)
+
+        merged_setbacks = setbacks_class(EXCL_H5, *setback_distance,
+                                         regulations_fpath=regs_fpath,
+                                         weights_calculation_upscale_factor=sf,
+                                         multiplier=100)
+        merged_layer = merged_setbacks.compute_setbacks(features_path,
+                                                        max_workers=1)
```

Comment:
I'm a little confused on where the actual merge request happens... Like you're not actually requesting a merge of two exclusion layers but instead just combining the setback distance input with an explicit regulations file? Otherwise looks good to me!

Reply:
Right, I think the title is a bit misleading (I essentially copied the issue request)

[Anthony had said](https://github.com/NREL/reVX/issues/146#issue-1262533179):  "Either add a flag to handle all of this in one go e.g., model existing ordinances and apply [generic ordinances] elsewhere. Or have a new method that assumes you've already created the necessary datasets and just want to merge them (less ideal)."

He said the second option is less ideal so I went with the first idea (except there was no need for a new flag value)

Reply:
Merging this so that Anthony can start using it, but let me know if you have any other questions/objections!